kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayClass.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.stream.array;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class ArrayClass implements ArrayOperations{
+
+List<Integer> numbers = new ArrayList<>();
+
+    @Override
+    public double getAverage() {
+        int listOfNumbers = IntStream.range(0, numbers.size())
+                .forEach(x -> System.out.println(x));
+        double avarageNumber = IntStream.range(0, numbers.size())
+                .average();
```

Comment:
tutaj sa 2 problemy
1) .average zwraca type OptionalDouble. Jest to typ ktory opakowuje wartosc double. Musisz go rozpakowac poprzez np `getAsDouble()` albo `orElse(0.0)`
O optionalach:
http://www.baeldung.com/java-optional

2) IntStream.range(0, numbers.size().average().orElse(0.0) zwroci srednia indeksu (0,number,size().
np dla wartosci numbers -> 0 5 10, zwroci 1 zamiast 5. Wynika to z tego ze zostania dodane elementy o wartosciach 0 + 1 + 2, zamiast 0 + 5 + 10